concert/app/src/functions/authorizer/index.py

```diff
@@ -1,36 +1,96 @@
 import base64
+import hashlib
 import json
 import os
+import time
+from jose import jwk, jwt
+from jose.utils import base64url_decode
+from urllib.request import urlopen
 
 API_ARN = os.environ.get("API_ARN")
+API_METHOD_CHOICE_MADE = os.environ.get("API_METHOD_CHOICE_MADE")
+API_METHOD_EVENT_STAGE_CHANGED = os.environ.get("API_METHOD_EVENT_STAGE_CHANGED")
+COGNITO_APP_CLIENT_ID = os.environ.get("COGNITO_APP_CLIENT_ID")
+COGNITO_GROUP_ADMIN = os.environ.get("COGNITO_GROUP_ADMIN")
+COGNITO_GROUP_VISUALIZATION = os.environ.get("COGNITO_GROUP_VISUALIZATION")
 COGNITO_USER_POOL_ID = os.environ.get("COGNITO_USER_POOL_ID")
-COGNITO_ADMIN_GROUP_NAME = os.environ.get("COGNITO_ADMIN_GROUP_NAME")
+COGNITO_REGION = COGNITO_USER_POOL_ID.split("_")[0]
 
-# TODO:
-# Verify JWT
-# Check that user is in the concert group
-# Write method-specific policy based on user groups (admin, vis, etc.)
+response = urlopen(
+    "https://cognito-idp.{}.amazonaws.com/{}/.well-known/jwks.json".format(
+        COGNITO_REGION, COGNITO_USER_POOL_ID
+    )
+)
+JWKS_KEYS = json.loads(response.read())["keys"]
 
+CHOICE_BUCKETS = [
+    "CHOICE_COLOR",
+    "CHOICE_COLOR",
```

Comment:
Duplication of line above. Should this be CHOICE_CHILLS?

Reply:
No. 50% chance color, 25% chance happiness, 25% agitation

Reply:
Ah. Got it.